apps/web/src/components/ClickableLogo/ClickableLogo.stories.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react';
+import ClickableLogo from './ClickableLogo';
+
+export default {
+  title: 'Components/ClickableLogo',
+  component: ClickableLogo,
+  argTypes: {
+    onClick: { action: 'clicked' },
+  },
+};
```

Comment:
You are missing the TypeScript for this story and that's why you have to define a `ClickableLogoArgs` type and why you're `Default.args` isn't working. 

So your stories should always have the same type as the component they represent, in this case `ClickableLogo.tsx`. But... out-of-the-box, `Storybook` doesn't know the types for `ClickableLogo`, so we've got to tell it. 

You're missing the `meta` and `storyFn` types that come from `Storybook`. Add those and you should be able to remove the `ClickableLogoArgs` type and everything should work.